ansible/idr-ebi-nfs.yml

```diff
@@ -36,6 +37,5 @@
 
 
   vars:
-    nfs_mountpoint: /nfs/bioimage
     nfs_filesets_dir: /nfs/bioimage/drop/complete
```

Comment:
What would you do about this variable? In the absence of `nfs_mountpoint` it still feels pretty specific to the current IDR deployment.

Reply:
It is very specific, but I couldn't immediately see how it fits together with https://github.com/openmicroscopy/management_tools/blob/master/idr/playbooks/idr-links.yml so left it as it is for now

Reply:
For reasons lost to the distant past these seem to be redundant. I've removed them.